tests/functional/parser/test_sns.py

```diff
@@ -103,3 +104,29 @@ def handle_sns_sqs_json_body(event: List[MySnsBusiness], _: LambdaContext):
 def test_handle_sns_sqs_trigger_event_json_body():  # noqa: F811
     event_dict = load_event("snsSqsEvent.json")
     handle_sns_sqs_json_body(event_dict, LambdaContext())
+
+
+def test_handle_sns_sqs_trigger_event_json_body_missing_signing_cert_url():
+    # GIVEN an event with a missing SigningCertURL
+    event_dict = load_event("snsSqsEvent.json")
+    payload = json.loads(event_dict["Records"][0]["body"])
+    assert payload.pop("SigningCertURL") is not None
+    event_dict["Records"][0]["body"] = json.dumps(payload)
+
+    # WHEN parsing the payload
+    # THEN raise a ValidationError error
+    with pytest.raises(ValidationError):
+        handle_sns_sqs_json_body(event_dict, LambdaContext())
```

Comment:
```suggestion
def test_handle_sns_sqs_trigger_event_json_body_missing_signing_cert_url():
    # GIVEN an event is tampered with a missing SigningCertURL
    event_dict = load_event("snsSqsEvent.json")
    payload = json.loads(event_dict["Records"][0]["body"])
    payload.pop("SigningCertURL")
    event_dict["Records"][0]["body"] = json.dumps(payload)

    # WHEN parsing the payload
    # THEN raise a ValidationError error
    with pytest.raises(ValidationError):
        handle_sns_sqs_json_body(event_dict, LambdaContext())
```

Added an explicit description on GIVEN that this is tampered, not the behaviour we'd expect from SNS. Also removed the `assert` as it'll only test whether we're changing the underlying fake event.

Reply:
@heitorlessa this is an interesting approach. Technically I think that Assert makes sense but the reason you ask to remove it is to make the code cleaner and without deviation from what should really be tested. We are informing the payload statically (snsSqsEvent.json) and we are not previously changing it. So, in this case we are sure that the command (payload.pop) will work.

Good catch and new thing learned! Btw, I would have done exactly like you @rubenfonseca lol.

Congrats for your first PR @rubenfonseca :rocket: :+1: